apps/server/src/services/cache.service.ts

```
@@ -0,0 +1,22 @@
+    }
+
+    static async setUserProfile(userId: string, profile: any) {
+        const key = `${this.PREFIX}user:${userId}`;
+        await redis.setex(key, this.TTL, JSON.stringify(profile));
+    }
+
+    static async invalidateUserProfile(userId: string) {
+        const key = `${this.PREFIX}user:${userId}`;
+        await redis.del(key);
+    }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)